src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -255,7 +255,7 @@ public void SarifErrorListItem_WhenFixHasRelativePath_UsesThatPath()
             };
 
             SarifErrorListItem item = MakeErrorListItem(result);
-
+            item.PopulateFixModelsIfNot();
```

Comment:
>PopulateFixModelsIfNot [](http://example.com/codeflow?start=17&length=22)

you could check if you have fixes before and after.
i would expect that if we don't call the method, the fix wouldn't be loaded